### PR TITLE
ADEN | change invisible high impact's HTML so it's the same on both platforms

### PR DESCRIPTION
**Pull request by bognix**
https://github.com/Wikia/mercury/pull/2734

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [Ad_Top.php#L5](https://github.com/Wikia/app/blob/716adc4d9c003dab6f15ee4b34ad45a3877a87fd/skins/oasis/modules/templates/Ad_Top.php#L5): Define a constant instead of duplicating this literal "Index" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [Ad_Top.php#L6](https://github.com/Wikia/app/blob/716adc4d9c003dab6f15ee4b34ad45a3877a87fd/skins/oasis/modules/templates/Ad_Top.php#L6): Define a constant instead of duplicating this literal "slotName" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [Ad_Top.php#L7](https://github.com/Wikia/app/blob/716adc4d9c003dab6f15ee4b34ad45a3877a87fd/skins/oasis/modules/templates/Ad_Top.php#L7): Define a constant instead of duplicating this literal "search" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [Ad_Top.php#L7](https://github.com/Wikia/app/blob/716adc4d9c003dab6f15ee4b34ad45a3877a87fd/skins/oasis/modules/templates/Ad_Top.php#L7): Define a constant instead of duplicating this literal "all_ads" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [Ad_Top.php#L7](https://github.com/Wikia/app/blob/716adc4d9c003dab6f15ee4b34ad45a3877a87fd/skins/oasis/modules/templates/Ad_Top.php#L7): Define a constant instead of duplicating this literal "homepage_logged" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [Ad_Top.php#L7](https://github.com/Wikia/app/blob/716adc4d9c003dab6f15ee4b34ad45a3877a87fd/skins/oasis/modules/templates/Ad_Top.php#L7): Define a constant instead of duplicating this literal "pageTypes" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)

**Comment by jacekjursza** [comment]
LGTM

File: skins/oasis/css/core/ads-invisible-high-impact.scss [hunk 1]
```diff
@@ -26,14 +26,24 @@ $z-high-impact: $zTop + 2;
 		width: 100%;
 	}
 
+	.top-bar {
+		background-color: rgba(0,0,0,0.5);
```
Comment (fraszczakszymon) [inline skins/oasis/css/core/ads-invisible-high-impact.scss]: `left: 0;`

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [Ad_Top.php#L5](https://github.com/Wikia/app/blob/b8bd5e621a5b978cbc61eb1637952e19da9ddab2/skins/oasis/modules/templates/Ad_Top.php#L5): Define a constant instead of duplicating this literal "Index" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [Ad_Top.php#L6](https://github.com/Wikia/app/blob/b8bd5e621a5b978cbc61eb1637952e19da9ddab2/skins/oasis/modules/templates/Ad_Top.php#L6): Define a constant instead of duplicating this literal "slotName" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [Ad_Top.php#L7](https://github.com/Wikia/app/blob/b8bd5e621a5b978cbc61eb1637952e19da9ddab2/skins/oasis/modules/templates/Ad_Top.php#L7): Define a constant instead of duplicating this literal "search" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [Ad_Top.php#L7](https://github.com/Wikia/app/blob/b8bd5e621a5b978cbc61eb1637952e19da9ddab2/skins/oasis/modules/templates/Ad_Top.php#L7): Define a constant instead of duplicating this literal "all_ads" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [Ad_Top.php#L7](https://github.com/Wikia/app/blob/b8bd5e621a5b978cbc61eb1637952e19da9ddab2/skins/oasis/modules/templates/Ad_Top.php#L7): Define a constant instead of duplicating this literal "homepage_logged" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [Ad_Top.php#L7](https://github.com/Wikia/app/blob/b8bd5e621a5b978cbc61eb1637952e19da9ddab2/skins/oasis/modules/templates/Ad_Top.php#L7): Define a constant instead of duplicating this literal "pageTypes" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)

**Comment by jacekjursza** [comment]
LGTM

**Pull request merged**